test_suite/cloud/test_aws.py

```diff
@@ -553,6 +559,7 @@ def test_dracut_conf_xen(self, host):
 
     @pytest.mark.pub
     @pytest.mark.run_on(['rhel'])
+    @pytest.mark.usefixtures('rhel_aws_marketplace_only')
```

Comment:
I though you need opposite.
rhui works on 1p(golden images) because they have billing code.
But does not work for pre-release testing for 3p(stratosphere) because they are not yet associated with the AWS marketplace product.

Reply:
Thanks for the comment. The new `rhel_aws_marketplace_only` fixture introduced in this PR actually excludes Strato images.
We don't need "Strato-only" tests, but quite the opposite. MAybe the name is not the best, and it is also redundant compared to the fixture added in conftest.py, so we may probably remove this. Waiting for more feedback from cloudx peers :)

Reply:
> MAybe the name is not the best

yeah; I was confused from the name and did not check the implementation :-)